src/atoms/listItem/documentation.stories.mdx

```diff
@@ -12,5 +13,12 @@ import { ListItem, IListItemProps } from '.';
 This component is only intended to be used with [Molecules/List](/?path=/docs/molecules-list).
 
 ### Theming
+<Canvas>
```

Comment:
there should not be examples of this here. Please add an example in the List documentation. Also this example doesn't have anything disabled, please be sure to add that.